x-pack/plugins/observability/server/saved_objects/slo.ts

```diff
@@ -53,6 +67,7 @@ export const slo: SavedObjectsType = {
       budgetingMethod: { type: 'keyword' },
       enabled: { type: 'boolean' },
       tags: { type: 'keyword' },
+      version: { type: 'long' },
```

Comment:
@kdelemme I am a bit confused what we store here. Why isn't the `target`, or `groupBy` stored here? I would have the same question for `timeWindow`, but for this I see we have a migration. Maybe we need a migration for the fields I mentioned? Unless the ` modelVersions` change that was added in this file, takes care of the missing fields. Please clarify.

Reply:
This file does not list all the SLO fields. That's because we don't expect to search on SLO Saved Object on every possible fields, and because it has been recommended to not map all fields here for performance reason (since these fields get added into the ".kibana*" index, and it is already pretty wild.

Technically we could remove pretty much all the fields here (well we can't remove fields when they have been added), but we don't really use them anymore (since the new search capability, using the summary index). 
But version is actually used (for searching the outdated SLO)